### DISCUSSION TITLE
Flood fails to add items from showrss.info feed

### DISCUSSION THREAD

**Discussion by jdarias** [jesec/flood#286] [Q&A]
I'm having trouble downloading items in a RSS feed from https://showrss.info 
I have set the rss but flood fails to add the items. It also fails when I try to add an item manually. In fact it is failing to add magnets from many popular sites. Help needed.

**Comment by jesec** [comment]
I will take a look. Note that Flood can't solve captcha, so some Cloudflare protected sites may not work. 

You may have to use Jackett and/or FlareSolverr .

**Reply by jesec** [reply]
I am not able to reproduce. The rule and feed appear to work properly for me.
